src/in_memory_example.rs

```diff
@@ -70,35 +69,24 @@ impl<const UID_LENGTH: usize> FindexInMemory<UID_LENGTH> {
     pub fn remove_location(&mut self, location: Location) {
         self.removed_locations.insert(location);
     }
-
-    pub fn set_check_progress_callback_next_keyword(&mut self, value: bool) {
-        self.check_progress_callback_next_keyword = value;
-    }
 }
 
 impl<const UID_LENGTH: usize> FindexCallbacks<ExampleError, UID_LENGTH>
     for FindexInMemory<UID_LENGTH>
 {
     async fn progress(
         &self,
-        results: &HashMap<Keyword, HashSet<IndexedValue>>,
+        _results: &HashMap<Keyword, HashSet<IndexedValue>>,
     ) -> Result<bool, ExampleError> {
-        if self.check_progress_callback_next_keyword {
-            let keyword = &Keyword::from("rob");
-            let results = results
-                .get(keyword)
-                .ok_or_else(|| {
-                    ExampleError(format!(
-                        "Cannot find keyword {keyword:?} in search results {results:?}"
-                    ))
-                })
-                .unwrap();
-            assert!(results.contains(&IndexedValue::NextKeyword(Keyword::from("robert"))));
-        }
```

Comment:
Did you add back this test somewhere else?

Reply:
No, I didn't find it that useful...

Reply:
It checks that the progress callback is working no? Without it we can remove the call to the progress callback?

Reply:
I have added it back.